server/handle_policy_eval.go

```diff
@@ -20,18 +20,17 @@ import (
 )
 
 type EvalRego struct {
```

Comment:
would it make sense to move all structs to a different package?

Reply:
Actually, If no complex business logic, putting the structs where they have been used is also a good idea.

Reply:
was only thinking to keep all data models in once place. This will help us in future to easy check the correct places instead of searching.